src/modules/groupPercentageMethods.ts

```diff
@@ -0,0 +1,33 @@
+import { TRPCError } from '@trpc/server';
+
+import { prisma } from '../utils/prisma';
+
+import { UpdateMembershipPercentage } from './percentageSchemas';
+import { tr } from './modules.i18n';
+import { userMethods } from './userMethods';
+
+export const groupPercentageMethods = {
```

Comment:
I think this should be in group methods. There won't be any more percenate-related methods.